app/lib/bk/compat/parsers/circleci/orbs/docker/install_tools.rb

```diff
@@ -34,6 +58,46 @@ def install_commands(version, install_dir)
             ]
           end
         end
+
+        def install_docker_compose_commands(version, install_dir)
+          platform = `uname -s`.strip.downcase
+          sys_arch = `uname -m`.strip.downcase
```

Comment:
the backticks would run when the conversion tool is run in - what one could assume would be - a dev's computer and not the same place where the steps will run so this would mean that there is a very likely architecture and OS mismatch